### DISCUSSION TITLE
Make it possible to share a gym

### DISCUSSION THREAD

**Discussion by lsolesen** [astashov/liftosaur#210] [Ideas]
I am teaching a lot of students in our school gym. It would be awesome, if I somehow could share the gym setup with the students, so they do not have to put in dumbbell weights, kettlebell weights, plates weight, trapbar weight and so on when I have already done the work for them.

That would make it easier for them to assign the equipment for the exercises and get proper roundings :)

**Comment by yrinkel** [comment]
Yes that would be good, also to not have to re-type all equipment when you and you partner go to the same gym. 

And pro trainers would obviously benefit from this greatly.
